airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml

```diff
@@ -28,11 +28,11 @@ clean-targets:         # directories to be removed by `dbt clean`
     - "dbt_modules"
 
 quoting:
-  database: true
+  database: false
 # Temporarily disabling the behavior of the ExtendedNameTransformer on table/schema names, see (issue #1785)
 # all schemas should be unquoted
   schema: false
-  identifier: true
+  identifier: false
```

Comment:
Oracle needs this to be false. Without it there is a problem to find tables and views. Maybe I can implement some logic to be specific to Oracle.

Reply:
Created function in `transform.py` `transform_dbt_project` to handle this per destination.